src/Transport/Creation/AzureServiceBusQueueCreator.cs

```diff
@@ -12,13 +12,13 @@
 
     class AzureServiceBusQueueCreator
     {
+        internal const int DefaultMaxDeliveryCountForNoImmediateRetries = int.MaxValue;
```

Comment:
Why is this maxvalue? won't this disable the dlq?

Reply:
Yes, disable accidental auto-forwarding to DLQ.